helper.linux.fish

```diff
@@ -52,6 +54,12 @@ end
 function pushCentosPackagingImage ; docker push $CENTOSPACKAGINGIMAGE ; end
 function pullCentosPackagingImage ; docker pull $CENTOSPACKAGINGIMAGE ; end
 
+function buildContainerDocumentation
```

Comment:
Why not "buildDocumentationImage"? see above

Reply:
The build script is guessed buildContainerDocumentation, I guess the function name was derived from that?

Reply:
Yes - It was there long before integrating the call into the rest of oskar. I agree that it should be named equally.